.github/workflows/coverage.yml

```diff
@@ -42,4 +42,4 @@ jobs:
           pytest --cov=networkx --runslow --doctest-modules --durations=20 --pyargs networkx
 
       - name: Upload to codecov
-        uses: codecov/codecov-action@v3
+        uses: codecov/codecov-action@v4
```

Comment:
```suggestion
        uses: codecov/codecov-action@v3
```

Reply:
Switching to v4 involves more work than just changing this line. It should be done by hand.